lib/ffmpeg/vaapi/encoder.py

```diff
@@ -33,7 +33,7 @@ def profile(self):
   def qp(self):
     def inner(qp):
       if self.codec in [Codec.VP8, Codec.VP9, Codec.AV1]:
-        return f" -global_quality {qp}"
+        return f" -q {qp}"
```

Comment:
You may remove this check, and use `-q {qp}` for all codecs, including h264, h265, mpeg2, vp8, vp9, av1.

Reply:
thx, but I still need some time to verify that this modification will not cause any Regression issues in function/quality/ testing.

Reply:
much simpler, thx.